src/Compilers/CSharp/Test/Symbol/DocumentationComments/TypeDocumentationCommentTests.cs

```diff
@@ -135,5 +135,62 @@ class A
             var comp = CreateStandardCompilation(text);
             Assert.Equal(0, comp.GetDiagnostics().Count());
         }
+
+        [Fact]
+        [WorkItem(19756, "https://github.com/dotnet/roslyn/issues/19756")]
+        public void GenericTypeWithOpenTypeArgument()
+        {
+            string source = @"
+/// <see cref=""C{U}.M()"" />
+class D { }
+
+class C<T>
+{
+    void M() { }
+}
+";
+            var comp = CreateCompilationWithMscorlibAndDocumentationComments(source);
+
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree);
+
+            var generic = tree.GetRoot().DescendantNodes(descendIntoChildren: n => true, descendIntoTrivia: true)
+                .OfType<QualifiedCrefSyntax>().Single();
+
+            Assert.Equal("C{U}.M()", generic.ToString());
+
+            var symbol = model.GetSymbolInfo(generic).Symbol;
+            var docId = symbol.GetDocumentationCommentId();
+            Assert.Equal("M:C{`0}.M", docId);
+        }
+
+        [Fact]
+        [WorkItem(19756, "https://github.com/dotnet/roslyn/issues/19756")]
+        public void GenericMethodWithOpenTypeArgument()
+        {
+            string source = @"
+
+class C
+{
+    static void M<T>() { }
+
+    /// <see cref=""C.M{U}()"" />
+    void M2() { }
+}
+";
+            var comp = CreateCompilationWithMscorlibAndDocumentationComments(source);
+
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree);
+
+            var generic = tree.GetRoot().DescendantNodes(descendIntoChildren: n => true, descendIntoTrivia: true)
+                .OfType<QualifiedCrefSyntax>().Single();
+
+            Assert.Equal("C.M{U}()", generic.ToString());
+
+            var symbol = model.GetSymbolInfo(generic).Symbol;
+            var docId = symbol.GetDocumentationCommentId();
+            Assert.Equal("M:C.M``1", docId);
```

Comment:
>Assert.Equal("M:C.M``1", docId); [](start = 12, length = 32)

This doesn't look right. What Id do we get from from symbol representing declaration of ```M<T>```?

Reply:
Ok, it looks like this is expected for methods, for them we only include arity of the method after the method name. However, it would be good to try scenarios when method's parameter refers to a type parameter. 

---
In reply to: [155956234](https://github.com/dotnet/roslyn/pull/23679#discussion_r155956234) [](ancestors = 155956234)